tests/bug01534.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Test for bug #1534: Segfault when exception thrown in a closure bound to class scope (fixed by #1665, bugs have same root cause)
+--INI--
+xdebug.default_enable = 1
```

Comment:
Please no space around the ` = ` (in all test files).

Reply:
Easy; done